tests/unit/ReviewConfirmDissolution.spec.ts

```diff
@@ -5,33 +5,41 @@ import { ReviewConfirmDissolution } from '@/views/Dissolutions'
 const reviewConfirmTestCases = [
   {
     entityType: 'CP',
-    headerTitle: 'Review and Confirm',
-    hasEffectiveDateTime: false
+    isStaff: true
+  },
+  {
+    entityType: 'CP',
+    isStaff: false
   },
   {
     entityType: 'BEN',
-    headerTitle: 'Review and Confirm',
```

Comment:
haha i think this was more of an 'example' than anything 😛

Reply:
Well, now it's evolved a bit :)